contractcourt/commit_sweep_resolver.go

```diff
@@ -5,10 +5,16 @@ import (
 	"fmt"
 	"io"
 
-	"github.com/lightningnetwork/lnd/input"
-
 	"github.com/btcsuite/btcd/wire"
+	"github.com/lightningnetwork/lnd/input"
 	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/sweep"
+)
+
+const (
+	// commitOutputConfTarget is the default confirmation target we'll use
+	// for sweeps of commit outputs that belong to us.
+	commitOutputConfTarget = 6
```

Comment:
I like it, explicit without a default. Later we could move those settings for different classes of inputs to a shared location possibly. Or add some logic around it to set the conf target based on the value.